contracts/token/ERC777/ERC777.sol

```diff
@@ -331,47 +330,47 @@ contract ERC777 is Context, IERC777, IERC20 {
      *
      * Requirements
      *
-     * - `account` cannot be the zero address.
-     * - if `account` is a contract, it must implement the {IERC777Recipient}
+     * - `to` cannot be the zero address.
+     * - if `to` is a contract, it must implement the {IERC777Recipient}
      * interface.
      */
     function _mint(
-        address account,
+        address to,
         uint256 amount,
-        bytes memory userData,
+        bytes memory data,
```

Comment:
Given that there is `operatorData` I think it makes sense for this to be `userData`.